README.md

```diff
@@ -1,21 +1,82 @@
-# SAGED
+# Linear and Nonlinear Signals
+This repo contains the code to reproduce the results of the manuscript "The Effects of Nonlinear Signal on Expression-Based Prediction Performance".
+In short, we compare linear and nonlinear models in multiple prediction tasks, and find that their predictive ability is roughly equivalent.
+Further, this similarity is despite the fact that predictive nonlinear signal exists in the data for each of the tasks.
 
-![Status Badge](https://github.com/greenelab/saged/workflows/PythonTests/badge.svg)
+![model comparison figure](https://raw.githubusercontent.com/greenelab/linear_signal/master/figures/full_signal_combined.svg)
 
-## Setup
-Before running scripts involving sex prediction, you need to download the Flynn et al. labels from [this link](https://figshare.com/s/985621c1705043421962) and put the results in the `saged/data` directory.
-Because of the settings on the figshare repo it isn't possible to incorporate that part of the data download into the Snakefile, otherwise I would.
+## Installation
+
+### Python dependencies
+The Python dependencies for this project are managed via [Conda](https://docs.conda.io/en/latest/miniconda.html).
+To install them and activate the environment, use the following commands in bash:
+
+``` bash
+conda env create --file environment.yml
+conda activate linear_models
+```
+
+### R setup
+The R dependencies for this project are managed via [Renv](https://rstudio.github.io/renv/articles/renv.html).
+To set up Renv for the repository, use the commands below within R while working in the `linear_signals` repo:
+
+``` R
+install.packages('renv')
+renv::init()
+renv::restore()
+```
+
+### Reproducing results
+The pipeline to download all the data, and produce all the results shown in the manuscript is managed by Snakemake.
```

Comment:
Do you want to give a rough estimate of how much memory this pipeline requires somewhere in this README? I'm assuming it's more than the typical laptop has (but maybe I'm wrong)

Reply:
Good call, I went ahead and added it!